api_testing/apis/orders_api.py

```diff
@@ -163,12 +159,14 @@ def create_order_with_multiple_suborders(self, no_suborders=3, suborders_fields=
             'testDatedateOption': {'year': test_date_arr[0], 'month': test_date_arr[1], 'day': test_date_arr[2]}
         }
         suborders = []
+        sub_order_dict = {}
```

Comment:
there is no need for it. please remove it.

Reply:
done